src/fidesops/service/connectors/query_config.py

```diff
@@ -396,6 +398,73 @@ def dry_run_query(self) -> Optional[str]:
         return None
 
 
+class MicrosoftSQLServerQueryConfig(SQLQueryConfig):
+    """
+    Generates SQL valid for SQLServer. This way of building queries should also work for every other connector,
+    but SQLServer is separated due to increased code complexity for building queries
+    """
+
+    def format_clause_for_query(
+        self, string_path: str, operator: str, operand: str
+    ) -> str:
+        """Returns clauses in a format they can be added into SQL queries."""
+        if operator == "IN":
+            return f"{string_path} IN ({operand})"
```

Comment:
Do we need a `:` before `{operand}` or will this already be there?

Reply:
it will already be there